### PR TITLE
Remove documentary comments/associated whitespace

### PR DESCRIPTION
**Pull request by tatyree**
I started to do this as part of the rubocop LineLength TODO. On
reflection, it seemed cleaner to do it on a separate branch as not all
of them were excessively long.  Nevertheless, we don't want them in the
codebase.  Where removal would result in an empty file, as with
config/initializers/backtrace_silencers.rb, I have left the opening
comment (almost always a warning to restart the server if the file
changes).

### CODE REVIEW

File: config/initializers/inflections.rb [hunk 1]
```diff
@@ -1,16 +1 @@
 # Be sure to restart your server when you modify this file.
```
Comment (SteveMarshall) [inline config/initializers/inflections.rb]: What about this one?

File: config/initializers/backtrace_silencers.rb [hunk 2]
```diff
@@ -1,7 +1 @@
 # Be sure to restart your server when you modify this file.
```
Comment (SteveMarshall) [inline config/initializers/backtrace_silencers.rb]: What about this?

**Comment by SteveMarshall** [comment]
Other than those two comments, this looks great. Remove all the things!

**Comment by tatyree** [comment]
@SteveMarshall from my opening comment: "Where removal would result in an empty file, as with
config/initializers/backtrace_silencers.rb, I have left the opening
comment..." :)

**Comment by SteveMarshall** [comment]
@tatyree D'oh! Is there any reason not to remove the files? Would it break anything?

**Comment by tatyree** [comment]
@SteveMarshall I'll try and see what happens.

**Comment by tatyree** [comment]
@SteveMarshall Looks GTG. Specs pass without them, removed and rebased into original commit.

**Comment by threedaymonk** [comment]
caedete eos!

**Pull request merged**